src/components/Select/Select.test.tsx

```diff
@@ -43,6 +53,16 @@ test('it renders the provided helpMessage', async () => {
   expect(helpMessage).toBeInTheDocument();
 });
 
+test('it renders the provided labelSecondary', async () => {
+  const props = getBaseProps();
+  const { findByText } = renderWithTheme(
+    <Select {...props} data-testid={testId} helpMessage="help!" />
+  );
+
+  const helpMessage = await findByText('help!');
+  expect(helpMessage).toBeInTheDocument();
+});
```

Comment:
It looks like this is maybe a duplicate/no-op test?

Reply:
🤦‍♂️, yes haha updating now